feature/quicksettings/src/main/java/com/google/jetpackcamera/feature/quicksettings/ui/QuickSettingsComponents.kt

```diff
@@ -241,30 +276,32 @@ fun QuickSettingUiItem(
     text: String,
     accessibilityText: String,
     onClick: () -> Unit,
-    isHighLighted: Boolean = false
+    isHighLighted: Boolean = false,
+    enabled: Boolean = true
 ) {
     Column(
         modifier =
         modifier
             .wrapContentSize()
             .padding(dimensionResource(id = R.dimen.quick_settings_ui_item_padding))
-            .clickable {
-                onClick()
-            },
+            .clickable(onClick = onClick, enabled = enabled),
         verticalArrangement = Arrangement.Center,
         horizontalAlignment = Alignment.CenterHorizontally
     ) {
-        val tint = if (isHighLighted) Color.Yellow else Color.White
-        Icon(
-            painter = painter,
-            contentDescription = accessibilityText,
-            tint = tint,
-            modifier =
-            Modifier
-                .size(dimensionResource(id = R.dimen.quick_settings_ui_item_icon_size))
-        )
+        val contentColor = (if (isHighLighted) Color.Yellow else Color.White).let {
```

Comment:
I recall you mentioned setting the alpha to `.38f` comes from the material guidelines. Might be helpful to mention in a comment why that _particular_ number was chosen

Reply:
Linked to material 3 guidelines.